deploy/template.yaml

```diff
@@ -320,6 +320,11 @@ Resources:
   AccessLogsBucket:
     Type: AWS::S3::Bucket
     #checkov:skip=CKV_AWS_18: This is the bucket where our access logs go and AWS advise not sending a bucket's access logs to itself.
+    Metadata:
+      cfn-lint:
+        config:
+          ignore_checks:
+            - W3045 # Using a legacy access control setup for now
```

Comment:
cc @blakeyp might be nice to fix this, but felt out of scope for now. AWS recommend using a bucket policy on the access logs instead of `LogDeliveryWrite` (which uses ACLs).